src/components/Participants/Participants.js

```diff
@@ -0,0 +1,208 @@
+import React, { useState, useEffect } from "react"; 
+import style from './Participants.module.css';
+
+import Input from '../Input/Input';
+import Button from '../Button/Button';
+
+function Participants(index) {
+  const [info, setInfo] = useState({
+    date: '', 
+    location: '', 
+    ammount: ''
+  });
+
+  const [host, setHost] = useState({
+    hostName: '',
+    email: ''
+  });
+  
+  const blankParticipant = { name: '', email: '' };
+  const [participants, setParticipants] = useState([
+    { ...blankParticipant },
+  ]);
+
+  const handleInfoChange = (event) => 
+    setInfo({...info, [event.target.name]: event.target.value,});
+
+  const handleHostChange = (event) => 
+    setHost({...host, [event.target.name]: event.target.value,});
+
+  const handleParticipantChange = (event) => {
+    const updatedParticipants = [...participants];
+    updatedParticipants[event.target.dataset.index][event.target.title] = event.target.value; 
+    setParticipants(updatedParticipants);
+  };
+
+  const handleAddParticipant = () => {
+    setParticipants([...participants, { ...blankParticipant }]);
+  };
+
+  const handleRemoveParticipant = index => (event) => {
+    event.preventDefault();
+    setParticipants(participants.filter((p, pindex) => index !== pindex));
+  };
+
+  const handleSubmit = () => {
+    console.log(`The party will be held on: ${info.date}, on the location: ${info.location}, the max ammount of money is: ${info.ammount} RSD`);
+    console.log(`The host is: ${host.hostName}`);
+    
+    const parti = participants.concat(host);
+
+    parti.sort(() => 0.5 - Math.random());
+    const pairs = [];
+  
+    while (parti.length >=2) {
+      const pair = [parti.pop(), parti.pop()];
+      console.log('Pair: ', pair);
+      pairs.push(pair);
+    }
+  
+    console.log('All pairs', pairs)
+  }
+  
+  useEffect(() => console.log(info));
+  useEffect(() => console.log(host));
+  useEffect(() => console.log(participants));
+
+
+  const participantId = `name-${index}`;
+  const emailId = `email-${index}`;
+
+  return(
+    <div>
+      <div className={style.form_block}>
+      <form className={style.info_form}>
+        <h1 className={style.add_your_participants}>Add your participants</h1>
+        <div className={style.party_info}>
+          <div className={style.party_info_desc}>
+            <p className={style.party_info_desc_date}>Date of your Secret Santa party</p>
```

Comment:
Consider placing label and input fields grouped because it will make code more readable. Similar as you did for participants